tasks/jsinspect.js

```diff
@@ -61,8 +61,41 @@ module.exports = function(grunt) {
       });
     }
 
+    function hookStdout(callback) {
+      var oldWrite = process.stdout.write;
+
+      process.stdout.write = (function(write) {
+        return function(string, encoding, fd) {
+          write.apply(process.stdout, arguments);
+          callback(string, encoding, fd);
+        };
+      })(process.stdout.write);
+
+      return function() {
+        process.stdout.write = oldWrite;
+      };
+    }
+
+    var unhook;
+    var stdoutFS;
+
+    if (typeof options.outputPath === 'string') {
+      stdoutFS = require('fs').createWriteStream(options.outputPath, {encoding: 'utf8', flags: 'w'});
```

Comment:
Personally I'm not a big fan of these inline `require` statements. But this is probably only my taste. :)